src/AspNet.Security.OAuth.Bilibili/BilibiliAuthenticationHandler.cs

```diff
@@ -145,6 +144,27 @@ protected override async Task<AuthenticationTicket> CreateTicketAsync(
         return new AuthenticationTicket(context.Principal!, context.Properties, Scheme.Name);
     }
 
+    private SortedList<string, string> BuildXBiliHeaders()
+    {
+        var list = new SortedList<string, string>(StringComparer.OrdinalIgnoreCase)
+        {
+            { "x-bili-accesskeyid", Options.ClientId },
+            { "x-bili-content-md5", "d41d8cd98f00b204e9800998ecf8427e" }, // It's a GET request so there's no content, so we send the MD5 hash of an empty string
+            { "x-bili-signature-method", "HMAC-SHA256" },
+            { "x-bili-signature-nonce", GenerateNonce() },
+            { "x-bili-signature-version", "2.0" },
+            { "x-bili-timestamp", TimeProvider.GetUtcNow().ToUnixTimeSeconds().ToString(CultureInfo.InvariantCulture) }
+        };
+        return list;
```

Comment:
Can just return it and remove the local, and if we're optimising then should pass the capacity.

```suggestion
        return new SortedList<string, string>(6, StringComparer.OrdinalIgnoreCase)
        {
            { "x-bili-accesskeyid", Options.ClientId },
            { "x-bili-content-md5", "d41d8cd98f00b204e9800998ecf8427e" }, // It's a GET request so there's no content, so we send the MD5 hash of an empty string
            { "x-bili-signature-method", "HMAC-SHA256" },
            { "x-bili-signature-nonce", GenerateNonce() },
            { "x-bili-signature-version", "2.0" },
            { "x-bili-timestamp", TimeProvider.GetUtcNow().ToUnixTimeSeconds().ToString(CultureInfo.InvariantCulture) }
        };
```